wails-frontend/src/utils/massaFormat.tsx

```diff
@@ -20,6 +22,10 @@ export function toMASS(num: string | number): number {
   return Number(toMAS(num));
 }
 
+export function toMASString(num: string): string {
+  return toMAS(num).toFixed(MASSA_SCALING_FACTOR).toString().replace(/\.?0+$/, '');
+}
+
 export function toNanoMASS(str: string): number {
   const formattedString = str?.replace(/[^0-9.-]/g, ''); // Remove non-numeric characters
```

Comment:
wtf is this ? :'(

Reply:
i propose you tackle this in another PR...

Reply:
toMASS, and toNanoMASS will be removed in another PR